documentation/modules/rn-2.6.adoc

```diff
@@ -316,6 +331,10 @@ For a complete list of all resolved issues in this release, see the list of link
 
 This release has the following known issues:
 
+.Warm migration and remote migration flows are impacted by multiple bugs
```

Comment:
* Do you also want an admonition adding in [Warm migrations docs](https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.6/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#warm-migration_mtv)?

Reply:
Can we make this a INFO or NOTE box?

Reply:
@fabiand . Yes, of course (makes sense).

 There are five admonition types:

* NOTE
* TIP
* IMPORTANT
* CAUTION
* WARNING

![image](https://github.com/user-attachments/assets/f82270eb-8ba0-402a-b728-572c1636c2ac)

Added as a WARNING, but it is a single attribute change. So which would you prefer?

Reply:
this looks to me.